### ISSUE TITLE
Support running without privileges

### ISSUE THREAD

**Issue by dlipovetsky** [platform9/cctl#169]
The cctl state is stored in `/etc` by default, but a different location does not require privileges. All privileged actions (invoking nodeadm, etcdadm) are invoked via ssh.

**Comment by jayunit100** [comment]
yeah fixed in the quickstart so it works , can we just do like ~/.cctl
